### ISSUE TITLE
Can it be applied to multiple regions?

### ISSUE THREAD

**Issue by chatii** [seeebiii/ses-verify-identities#212]
I usually use ap-northeast-1 region for AWS development.
Since I need Cognito this time and I can't use SES in ap-northeast-1 to send emails via SES, I will set up SES in us-east-1.

The problem is that we want to set up the SES in ap-northeast-1 with the same domain name.
This is a way to avoid over pacific responses.

I kept trying and trying, but I encountered the following error.

> Error: There is already a Construct with name 'SesNotificationTopic' in VerifySesDomain 

Probably because of the duplicate ID of the Topic being created in this line

https://github.com/seeebiii/ses-verify-identities/blob/93a60750e4ea90b4aff9d76e7968f13a06ee8250/src/verify-ses-domain.ts#L177

Is there any way to work around this?

**Comment by ChristopheBougere** [comment]
I'm having a kinda similar issue right now: I'm using Cognito, and I need to validate an email from `us-east-1` for a stack that is in another region.

I would like the `VerifySesEmailAddress` construct to expose a `region` prop, that could be passed to the [AwsSdkCall construct](https://docs.aws.amazon.com/cdk/api/latest/docs/@aws-cdk_custom-resources.AwsSdkCall.html).
I could draft a PR if that makes sense @seeebiii 

Maybe that would help with your issue?

**Comment by seeebiii** [comment]
It looks like there are two separate problems here. Correct me if I'm wrong 🙂  

@chatii Do you deploy multiple stacks using `ses-verify-identities` in the same region? If so, you could reuse an existing topic using `notificationTopic` property of `VerifySesDomain`. Would that be sufficient?

@ChristopheBougere That sounds like a good addition! Feel free to create a PR, I'm happy to merge it. Please consider using [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/).

**Comment by chatii** [comment]
@seeebiii 
I see, I can work around it by referring to an existing SNS topic or a topic I created separately.
This seems to be a workaround for my problem. Thanks!

As a side note, when I created the SES manually on the AWS console, I was able to create it with "No SNS topic" and was trying to replicate that in the CDK.
I understood that it was `createTopic Or UseExisting` because it was a necessary item.
